public/config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
we have moved this file location and is not needed

Reply:
removed the file